resources/operators/mbp_client/README.md

```diff
@@ -0,0 +1,39 @@
+# How to use the MBP client
+
+The MBP client corresponds to a python script named [`mbp_client.py`](mbp_client.py). This script contains the logic to connect and communicated with the MBP. It abstracts a MQTT client and further configuration steps.
+
+To create your **own** python-based operator, you just need to import the MBP client in your customized python script:
```

Comment:
Talking about executing the operator manually, isn't it necessary to talk about the  `MBP.properties` file? As I understood, this file is automatically created by MBP when we execute the operator on the platform. However, if you don't execute the operator in this way, you need to create the file by yourself and also update its properties. 
But it is just as I understood haha! I may be mixing things up.

Reply:
The `MBP.properties` is created automatically during the deployment of operators. But if users do not deploy an operator through the MBP UI or REST API, what we do **not** recommend, starting the operator will not work without the `MBP.properties` file. 
I will add a hint to the `MBP.properties` for the case users choose to not use the MBP UI to deploy the operator.